newIDE/app/src/InAppTutorial/InAppTutorialTooltipDisplayer.js

```diff
@@ -215,7 +214,6 @@ const TooltipHeader = ({
   paletteType,
   progress,
   showFoldButton,
-  showQuitButton,
```

Comment:
As the flow is more restrictive now, I went with the decision to always show the quit button, so the user can always decide to quit, especially if there's a misconfigured step we didn't catch.